scriptler/findRunningPipelineJobs.groovy

```diff
@@ -0,0 +1,32 @@
+/*** BEGIN META {
+  "name" : "Find running pipeline jobs.",
+  "comment" : "Find all pipeline jobs running for at least a given amount of time (delay).",
+  "parameters" : [],
+  "core": "1.612.1",
+  "authors" : [
+    { name : "Arnaud Héritier" }
+  ]
+} END META**/
+
+import groovy.time.*
+
+def abortJobs = false
+
+use(TimeCategory)  {
+  
+  def delay = 1.day
+  
+  println "List of running jobs : "
+  Jenkins.instance.getAllItems(org.jenkinsci.plugins.workflow.job.WorkflowJob).each{
+    job -> job.builds.findAll{it.isBuilding() && new Date(it.startTimeInMillis) < ( new Date() - delay ) }.each{
```

Comment:
This will cycle through all jobs regardless of their build status (per run).  It would be a little more efficient to only search builds where `job.isBuilding()`.